docs/configuration.md

```diff
@@ -2483,6 +2483,7 @@ Spark subsystems.
     <code>spark.streaming.receiver.maxRate</code> and <code>spark.streaming.kafka.maxRatePerPartition</code>
```

Comment:
The version is not set in the code.

Reply:
Thanks. I will document them in a separate PR.

Reply:
The comment here is to add version(....) to the code which belongs to this PR...

Reply:
OK. I revert temporarily.

Reply:
That said the added version in the `md` file is fine and we shouldn't revert it (in order to have a complete config list with versions). The suggestion is to add the version in the code like this:
```
  private[spark] val MAX_RATE_PER_PARTITION =
    ConfigBuilder("spark.streaming.kafka.maxRatePerPartition")
    .version("1.3.0")
    .longConf
    .createWithDefault(0)
```

Reply:
> spark.streaming.kafka.maxRatePerPartition

This config exists in `external/kafka-0-10`

Reply:
@beliefer, I think you had to clarify you're working on this for each file as the logical group. Seems like that missing context is causing the miscommunication here with @gaborgsomogyi.

Reply:
Thanks for your explanation.

Reply:
Now I see the point and knowing this context LGTM. Thanks.